js/app.js

```diff
@@ -61,3 +65,61 @@ bookList.addEventListener('click', (e) => {
     bookFun.removeBook(targetEl);
   }
 });
+
+// Date and time
+const clockElement = document.getElementById('clock');
+
+function clock() {
+  clockElement.textContent = new Date().toString();
+}
+
+setInterval(clock, 1000);
+
+// nav bar
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Please you should not disable any eslint in your code. please make sure that you remove this line and fix what eslint is saying :blush: